src/main/java/com/mateacademy/spring_mvc/SpringMvcApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.mateacademy.spring_mvc;
```

Comment:
no underscores in package names 
[Package names](https://mate-academy.github.io/style-guides/java/java.html#s5.2.1-package-names)